HPXMLtoOpenStudio/tests/test_hvac_sizing.rb

```diff
@@ -541,27 +566,36 @@ def test_manual_j_residences
     assert_in_delta(1391, hpxml_bldg.hvac_plant.cdl_lat_infil, block_tol_btuh)
     assert_in_delta(0, hpxml_bldg.hvac_plant.cdl_lat_vent, block_tol_btuh)
     assert_in_delta(800, hpxml_bldg.hvac_plant.cdl_lat_intgains, block_tol_btuh)
+    # eyeball observation from figure 14-3
+    block_aed = [2500, 3200, 3900, 4200, 4600, 7500, 10600, 13900, 15200, 16100, 12700, 4100]
+    hpxml_bldg.hvac_plant.cdl_sens_aed_curve.split(', ').map { |s| s.to_f }.each_with_index do |aed_curve_value, i|
+      assert_in_delta(block_aed[i], aed_curve_value, [block_aed[i] * space_tol_frac, block_tol_btuh].max)
+    end
     space_load_results = {
       ['living', 'dining'] => [2534, 7176],
       ['kitchen', 'utility', 'entry'] => [773, 2121],
-      ['bedroom_3'] => [1550, 3572],
+      # ['bedroom_3'] => [1550, 3572], # Bedroom 3 for some reason has much greater heating loads than bedroom 2, which is of similar layout, exclude it for now
```

Comment:
I couldn't figure out why the bedroom 3 heating loads are different from bedroom 2. They're almost the same layout and the same load components, internal gains are different but shouldn't apply to heating load calculation.

Reply:
All your comments sound reasonable. There's only so much we can do with the examples. We are mostly looking for obvious outliers that may suggest we have a bug or deficiency. I think we've exhausted what we can learn from the examples and what we can build tests around, so we should call this effort complete and move on. 👍